src/main/pages/che-7/end-user-guide/assembly_the-developer-workspace-button.adoc

```diff
@@ -0,0 +1,39 @@
+---
+title: The Developer Workspace Button
+keywords:
+tags: []
+sidebar: che_7_docs
+permalink: che-7/the-developer-workspace-button
+summary:
+---
+
+:page-liquid:
+:parent-context-of-the-developer-workspace-button: {context}
+
+[id="the-developer-workspace-button_{context}"]
+= The Developer Workspace Button
+
+:context: the-developer-workspace-button
+
+It is common for code repositories to display badges in their `README` files that describe various aspects of the project, such as its test coverage, latest build status, link to documentation, and other things. For example, the {prod-short} GitHub `README` has the following badges:
+
+image::workspace_button/eclipse_che_badges.png[link="{imagesdir}/workspace_button/eclipse_che_badges.png"]
+
+To make the process of using {prod-short} easier, the following badge is available:
+
+image::https://che.openshift.io/factory/resources/factory-contribute.svg[Factory Badge]
+
+You can include this badge and link it to a {prod-short} instance to quickly open a developer workspace.
+
+
+== Including the Developer Workspace Button in a `README`
+
+=== Procedure
+
+Add a link to your repository in the project `README`.  Using a GitHub repository as an example:
+
+[source,shell]
+----
+[![Contribute](https://che.openshift.io/factory/resources/factory-contribute.svg)](https://che.openshift.io/f?url=https://github.com/org/repository)
```

Comment:
Perhaps we should use `che.openshift.io` as an example and list general instructions for adding the button? You should be able to replace `https://che.openshift.io` with whatever URI your Che server is deployed on.

Reply:
That’s correct.  This is an oversight on my part